### PR TITLE
Merge Topic / Purpose and update copy

### PR DESCRIPTION
**Pull request by rgarner**
# [Merge Topic / Purpose and update copy](https://trello.com/c/YIQw4isH/208-3-merge-topic-purpose-and-update-copy)

> As a researcher
> I want to provide topic and purpose in a clear and easy to understand way
> So that research participants know what is the aim of my research

screen shot 2017-11-02 at 10 38 51

### CODE REVIEW

**Comment by rgarner** [comment]
@Jesskyri @joelcarr @jeddy3 not sure whether we need the text areas to be actually inline/text *fields*

**Comment by Jesskyri** [comment]
@rgarner @joelcarr shall we use this placeholder text? 
*topic* Barnardo’s is doing research to learn about [how young people think of consent and research] *purpose* so that we can [talk to them in a way that they understand.]

**Comment by rgarner** [comment]
@sampling ah yes, placeholder text, thank you for the reminder :)

**Comment by joelcarr** [comment]
@rgarner they should be text fields as the desired outcome shouldn't have any formatting

**Comment by rgarner** [comment]
Thanks @joelcarr, will address at same time as placeholders

File: app/views/research_sessions/purpose.html.erb [hunk 1]
```diff
@@ -3,14 +3,6 @@
   <%= render 'error_summary' %>
```
Comment (joelcarr) [inline app/views/research_sessions/purpose.html.erb]: We should delete the whole file as it is no longer being used.

Comment (rgarner) [reply app/views/research_sessions/purpose.html.erb]: Oh yes, good spot

File: app/views/research_sessions/preview.html.erb [hunk 2]
```diff
@@ -24,34 +24,27 @@
 
 <div class="print-area js-print-area" id="info-sheets">
   <section>
-    <h2 class="heading-medium">About this research project</h2>
-    <p class="super-text">This information sheet will tell you more about the
-    research so that you can decide whether or not <%= you_or_your_child %> would like to take part.</p>
-  </section>
-
-  <section>
```
Comment (rgarner) [inline app/views/research_sessions/preview.html.erb]: @joelcarr does this look like a mistaken removal to you?

**Pull request merged**